pipelines/azurepipeline/code/deploy/environment.yml

```diff
@@ -16,4 +16,5 @@ dependencies:
   - numpy
   - tensorflow==2.0.0-alpha0
```

Comment:
Tensorflow 2 isn't Alpha anymore. How about instead we just change this line to:
`- tensorflow~=2`